spring-webflux/src/main/java/org/springframework/web/reactive/result/view/freemarker/FreeMarkerConfigurer.java

```diff
@@ -59,6 +57,7 @@
  * Note: Spring's FreeMarker support requires FreeMarker 2.3 or higher.
  *
  * @author Rossen Stoyanchev
+ * @author Issam El-atif
```

Comment:
```suggestion
```

We don't add ourselves as authors of a class if the only change we make is deletion of a TODO. So please revert this change.

Reply:
Done. Got it :)